semgrep/semgrep/core_runner.py

```diff
@@ -283,13 +281,12 @@ def _extract_core_output(
         # If semgrep-core prints anything on stderr when running with default
         # flags, it's a bug that should be fixed in semgrep-core.
         #
-        if semgrep_error_output != "":
-            name = f"[rule '{rule.id}']"
-            logger.info(
-                f"--- semgrep-core stderr {name} ---\n"
-                f"{semgrep_error_output}"
-                f"--- end semgrep-core stderr {name} ---"
-            )
+        name = f"[rule '{rule.id}']"
+        logger.debug(
```

Comment:
Logging handler in util.py handles whether or not a debug level log here gets printed to stderr